eng/dockerfile-templates/sdk/Dockerfile

```diff
@@ -29,6 +29,9 @@ RUN mkdir "%ProgramFiles%\NuGet\latest" `
     && curl -fSLo "%ProgramFiles%\NuGet\nuget.exe" https://dist.nuget.org/win-x86-commandline/v%NUGET_VERSION%/nuget.exe `
     && mklink "%ProgramFiles%\NuGet\latest\nuget.exe" "%ProgramFiles%\NuGet\nuget.exe"
 
+# Do not generate certificate
```

Comment:
At some point, I think we should consider a pattern where the ENVs are defined in a single place as done in the .NET images.  The primary goal is to improve the readability of discovering what ENVs are defined.